src/modules/admin/components/Permissions/Permissions.tsx

```diff
@@ -11,36 +15,58 @@ const MSG = defineMessages({
     id: 'dashboard.Permissions.title',
     defaultMessage: 'Permissions',
   },
+  addRole: {
+    id: 'dashboard.Permissions.addRole',
+    defaultMessage: 'Add Role',
+  },
 });
 
 interface Props {
   colonyAddress: Address;
+  openDialog: (
+    dialogName: string,
+    dialogProps?: Record<string, any>,
+  ) => DialogType;
 }
 
 const displayName = 'admin.Permissions';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars,no-unused-vars
-const Permissions = ({ colonyAddress }: Props) => (
-  <div className={styles.main}>
-    <main>
-      <div className={styles.titleContainer}>
-        <Heading
-          text={MSG.title}
-          appearance={{ size: 'medium', theme: 'dark' }}
-        />
-      </div>
-      <p>Placeholder for main</p>
-    </main>
-    <aside className={styles.sidebar}>
-      <ul>
-        <li>
-          <p>Placeholder for sidebar</p>
-        </li>
-      </ul>
-    </aside>
-  </div>
-);
+// eslint-disable-next-line no-unused-vars
+const Permissions = ({ colonyAddress, openDialog }: Props) => {
+  const handleEditPermissions = useCallback(
+    () =>
+      openDialog('ColonyPermissionEditDialog', {
+        colonyAddress,
+        domain: { name: 'Cooking', id: 5 },
```

Comment:
This `domain: { name: 'Cooking', id: 5 },` is just hardcoded until the Permissions screen is ready and merged. Will replace with real values before merging.